JS tasks/task cash/script.js

```diff
@@ -0,0 +1,37 @@
+const getBtn = document.querySelector('.get-btn');
```

Comment:
The ATM doesn't work as expected. You should have a limited number of money and show errors if user try withdraw more money or the amount of money doesn't match to nominals inside. Also user has to have limited number of attempt because when user withdraw the money it must affect an initial `limits` object so the number of used nominals must decrease

Reply:
fixed